security/s2a/internal/crypter/expander_test.go

```diff
@@ -7,59 +7,41 @@ import (
 	"testing"
 )
 
-type hkdfTestVector struct {
-	comment        string
-	prk, info, okm []byte
-	length         int
-}
-
-func TestExpanderExpansionInvalidLength(t *testing.T) {
-	tc := hkdfTestVector{
-		comment: "invalid length",
-		prk:     testutil.Dehex("07770932df0ddc3f0dc47bba6390b6c73bb50f9c3122ec844ad7c2b3e5"),
-		info:    testutil.Dehex("f0f1f2f3f4f5f6f7f8f9"),
-		length:  10000,
-	}
-
-	expander := newDefaultHKDFExpander()
-	got, err := expander.expand(sha256.New, tc.prk, tc.info, tc.length)
-	if err == nil {
-		t.Errorf("expand(sha256.New, %v, %v, %v) = %v, expected error but no error returned", tc.prk, tc.info, tc.length, got)
-	}
-}
-
-func TestExpanderExpansion(t *testing.T) {
-	// Test vectors were taken from https://tools.ietf.org/html/rfc5869.
-	for _, tc := range []hkdfTestVector{
+func TestExpand(t *testing.T) {
+	// The following test vectors were taken from
+	// https://tools.ietf.org/html/rfc5869. Note that they have been
+	// slightly modified to fit our interface and implementation.
```

Comment:
"slightly modified" is a big vague and for test vectors taken from the RFC, it's a bit scary :)

I suggest mention in the comment how the test vectors are exactly modified.

Reply:
Done.